pkg/core/helper_test.go

```diff
@@ -93,17 +93,18 @@ func (bc *Blockchain) genBlocks(n int) ([]*block.Block, error) {
 	lastHash := bc.topBlock.Load().(*block.Block).Hash()
 	lastIndex := bc.topBlock.Load().(*block.Block).Index
 	for i := 0; i < n; i++ {
-		minerTx := transaction.NewMinerTXWithNonce(uint32(1234 + i))
-		minerTx.ValidUntilBlock = lastIndex + uint32(i) + 1
-		err := addSender(minerTx)
+		tx := transaction.NewContractTX()
```

Comment:
I think we can issue blocks without any transactions here now.

Reply:
Is empty block valid?

Reply:
In Neo 3 it sure is.